exercise.wwwapi/Endpoints/StudentEndpoint.cs

```diff
@@ -0,0 +1,51 @@
+﻿using exercise.wwwapi.Repository;
+using Microsoft.AspNetCore.Mvc;
+
+namespace exercise.wwwapi.Endpoints
+{
+    public static class StudentEndpoint
+    {
+        public static void ConfigureStudentEndpoint(this WebApplication app)
+        {
+            var studentGroup = app.MapGroup("student");
+
+            studentGroup.MapGet("/students", GetStudents);
+            studentGroup.MapGet("/students/{FirstName}", GetStudent);
+            studentGroup.MapPut("/students/{FirstName}", UpdateStudent);
+            studentGroup.MapDelete("/students/{FirstName}", DeleteStudent);
```

Comment:
Students in the url and endpoints its called student

Reply:
fixed